flink-kubernetes-webhook/src/main/java/org/apache/flink/kubernetes/operator/admission/AdmissionHandler.java

```diff
@@ -62,7 +62,7 @@
 public class AdmissionHandler extends SimpleChannelInboundHandler<HttpRequest> {
     private static final Logger LOG = LoggerFactory.getLogger(AdmissionHandler.class);
     private static final ObjectMapper objectMapper = new ObjectMapper();
-    protected static final String VALIDATE_REQUEST_PATH = "/validate";
+    protected static final String VALIDATE_REQUEST_PATH = "/validateDeployment";
```

Comment:
I think you need to adjust the webhook template in the helm chart if you are changing this

Reply:
It's affected by the IDE refactor, reverted.